setup.go

```diff
@@ -340,6 +348,12 @@ func configFromEnv() Config {
 		}
 	}
 
+	if noExplicitFormat &&
+		(!cfg.Stdout || !isTerm(os.Stdout)) &&
+		(!cfg.Stderr || !isTerm(os.Stderr)) {
+		cfg.Format = PlaintextOutput
+	}
```

Comment:
Why do this here instead of on line 296?

Reply:
@Stebalien because this **later** hunk can change whether `cfg.Stdout` / `cfg.Stderr` are set... ?
https://github.com/ipfs/go-log/blob/61ec4a4d87eeea73b9c60ef10c1c93a3b1e836e5/setup.go#L331-L349

Reply:
Hm. I'd have just moved the switch statement down in that case, but it works either way.